rotkehlchen/tests/api/test_ethereum_transactions.py

```diff
@@ -1474,3 +1475,61 @@ def test_count_transactions_missing_decoding(rotkehlchen_api_server: 'APIServer'
         'ethereum': {'undecoded': 1, 'total': 2},
         'optimism': {'undecoded': 1, 'total': 1},
     }
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('have_decoders', [True])
+@pytest.mark.parametrize('ethereum_accounts', [['0xc37b40ABdB939635068d3c5f13E7faF686F03B65']])
+@pytest.mark.parametrize('should_mock_price_queries', [True])
+@pytest.mark.parametrize('default_mock_price_value', [ONE])
+def test_repulling_transaction_with_internal_txs(rotkehlchen_api_server: 'APIServer'):
+    """Check that re-decoding a transaction that has internal ETH transfers correctly
+    repulls those"""
+    tx_hash = deserialize_evm_tx_hash('0x4ea72ae535e32d5edc543a9ace5f736c7037cc63e4088de38511297c764049b5')  # noqa: E501
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    database = rotki.data.db
+    dbevents = DBHistoryEvents(database)
+    ethereum_inquirer = rotki.chains_aggregator.ethereum
+    get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer.node_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+
+    # query the prices so when we later compare the events the price doesn't affect.
+    # Prices are mocked
```

Comment:
```suggestion
    # query the prices so when we later compare the events the price doesn't affect.
    # Prices are mocked
```

The first line here makes no sense. What do you mean?